### ISSUE TITLE
use OpenSeadragon fullscreen?

### ISSUE THREAD

**Issue by edsu** [umd-mith/sga#327]
Now that we are using OpenSeadragon for the image pan/zoom I think it would be super to have the fullscreen image actually be fullscreen. This would make for a really nice pinch/zoom drag experience on an iPad or other tablet.

**Comment by raffazizzi** [comment]
Yeah to really do this we should create a custom navigation box for full screen only. It shouldn't be too hard and it could be based on the icons we already have.

**Comment by fraistat** [comment]
That would be really great! Thanks for the suggestion, Ed!--Neil

On Mon, Jun 15, 2015 at 8:12 PM, Raffaele Viglianti <
notifications@github.com> wrote:

> Yeah to really do this we should create a custom navigation box for full
> screen only. It shouldn't be too hard and it could be based on the icons we
> already have.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/umd-mith/sga/issues/327#issuecomment-112242226>.
>



-- 
Neil Fraistat
Professor of English & Director
Maryland Institute for Technology in the Humanities (MITH)
0301 Hornbake Library North
University of Maryland
301-405-5896 or 301-314-7111 (fax)
http://www.mith.umd.edu/
https://twitter.com/fraistat
